### ISSUE TITLE
Authentication with blueocean?

### ISSUE THREAD

**Issue by jackrvaughan** [issc29/jenkins-deployment-api#3]
Thanks for showcasing this example. 

For the authentication required for the http request to the github deployments API, I see you have `issc29-GH` referenced. What does that represent? 

Is there a way to use the blueocean authentication I already have with github?

**Comment by issc29** [comment]
Hey @jackrvaughan! the authentication in http request plugin represents a credential from the [Credential plugin](https://plugins.jenkins.io/credentials). Also check out the documentation for [HTTP Request Plugin](https://plugins.jenkins.io/http_request). You may be able to use your Blue Ocean credentials depending on the format, but you will need to find out the ID of the credentials from the Credential Plugin.

**Comment by lenisha** [comment]
@issc29 credential does not work if github org is enabled for 2FA, is there an option to send a token?

**Comment by issc29** [comment]
@lenisha yes, you should be able to use a token. For the password in Credentials plugin just use the token.
